cartridges/int_klaviyo_core/cartridge/scripts/klaviyo/eventData/addedToCart.js

```diff
@@ -0,0 +1,127 @@
+                if (basketProduct.variant) {
+                    primaryCategory = (basketProduct.masterProduct.primaryCategory) ? basketProduct.masterProduct.primaryCategory.displayName : '';
+                } else {
+                    primaryCategory = (basketProduct.primaryCategory) ? basketProduct.primaryCategory.displayName : '';
+                }
+                var imageSizeOfProduct = null;
+                if (KLImageSize && basketProduct.getImage(KLImageSize)) {
+                    imageSizeOfProduct = basketProduct.getImage(KLImageSize).getAbsURL().toString();
+                }
+
+                var categories = [];
+                var catProduct = (basketProduct.variant) ? basketProduct.masterProduct : basketProduct; // from orig klav code, always use master for finding cats
+                for(var i = 0, len = catProduct.categoryAssignments.length; i < len; i++) {
+                    categories.push(catProduct.categoryAssignments[i].category.displayName);
+                }
+
+                var currentLineItem = {
+                    productID       : currentProductID,
+                    productName     : basketProduct.name,
+                    productImageURL : imageSizeOfProduct,
+                    productPageURL  : URLUtils.https('Product-Show', 'pid', currentProductID).toString(),
+                    productUPC                : basketProduct.UPC,
+                    viewedProductAvailability : basketProduct.availabilityModel.availability,
+                    categories                : categories, // was createCategories(basketProduct) in orig, check that my output from categories above matches expected output
+                    primaryCategory           : primaryCategory
+                };
+
+                if(!basketProduct.master && 'masterProduct' in basketProduct) {
+                    currentLineItem.masterProductID = basketProduct.masterProduct.ID;
+                }
+
+                var priceData = klaviyoUtils.priceCheck(lineItem, basketProduct);
+                currentLineItem.price = priceData.purchasePrice;
+                currentLineItem.priceValue = priceData.purchasePriceValue;
+                currentLineItem.originalPrice = priceData.originalPrice;
+                currentLineItem.originalPriceValue = priceData.originalPriceValue;
+
+                selectedOptions = lineItem && lineItem.optionProductLineItems ? klaviyoUtils.captureProductOptions(lineItem.optionProductLineItems) : null;
+                if (selectedOptions && selectedOptions.length) {
+                    currentLineItem.productOptions = selectedOptions;
+                }
+
+                if (lineItem.bundledProductLineItem || lineItem.bundledProductLineItems.length) {
+                    var prodBundle = klaviyoUtils.captureProductBundles(lineItem.bundledProductLineItems);
+                    currentLineItem['Is Product Bundle'] = prodBundle.isProdBundle;
```

Comment:
If our past behavior didn't have keys with spaces, can we make these camel-case?

Reply:
decision: leave as is, no changes required